### PR TITLE
[#166761716] Added component for the organization logo in the list of services

### PR DESCRIPTION
**Pull request by Alex193a**
Added the organization logo to the list of services.

screen logo

Edit:

 The logo will appear in this way:

This PR now only includes the logic needed to add the organization's logo.
The UI will be updated in the PR #1158

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#166761716](https://www.pivotaltracker.com/story/show/166761716): Come CIT nella lista servizi voglio vedere anche il logo dell'ente

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Alex193a** [comment]
@matteodesanti Organization logos have a padding that causes a discrepancy between logo and service name, this makes it impossible to align all logos in the same way. The padding is not the same in all images.

**Comment by matteodesanti** [comment]
The available area for the logos should be 32x32px.
I meant the vertical alignment of this area and the relative label.
The label should be vertically centered.

**Comment by Alex193a** [comment]
@matteodesanti that misalignment will be fixed in another PR ([story #166792020](https://www.pivotaltracker.com/n/projects/2048617/stories/166792020) which will bring the restyling of the list).
Is the misalignment between logo and service name due to padding in logo images acceptable?
According to the [screen on invision](https://projects.invisionapp.com/share/43SKBSFCZY8#/screens) it should be aligned but considering the extra padding in the images is not possible.

File: ts/components/services/OrganizationLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { ImageURISource, StyleSheet } from "react-native";
+import { MultiImage } from "../ui/MultiImage";
+
+type Props = {
+  logoUri: ReadonlyArray<number | ImageURISource>;
```
[the remark this answers is not in the document]
Comment (Alex193a) [reply ts/components/services/OrganizationLogo.tsx]: Done. [1793018](https://github.com/teamdigitale/io-app/pull/1155/commits/1793018a89875df38243aa07ea428a73a27a839b)

**Comment by Alex193a** [comment]
@fpersico The story title has been renamed to " Added component for the organization logo in the list of services".
The branch has been merged with the master and the conflicts should be resolved.

File: patches/react-native-pull+2.0.2.patch [hunk 2]
```diff
@@ -4,199 +4,199 @@ index 0000000..f72e93a
 --- /dev/null
```
Comment (fpersico) [inline patches/react-native-pull+2.0.2.patch]: Why you are changing this file?

Comment (Alex193a) [reply patches/react-native-pull+2.0.2.patch]: It's caused by the merge of the branch with the master made with Github Desktop, I'm trying to fix it.

Comment (Alex193a) [reply patches/react-native-pull+2.0.2.patch]: @fpersico Fixed!

**Pull request merged**